google/cloud/bigtable/mutations.h

```diff
@@ -215,49 +252,52 @@ Mutation DeleteFromRow();
 class SingleRowMutation {
  public:
   /// Create an empty mutation.
-  explicit SingleRowMutation(std::string row_key)
-      : row_key_(std::move(row_key)) {}
+  template <typename RowKey>
+  explicit SingleRowMutation(RowKey row_key) : request_() {
```

Comment:
RowKey -> RowKey&&

Also, in several other places in this file, lines: 262, 272

Reply:
Missed those, darn, thanks for checking after me.